src/lib.rs

```diff
@@ -540,6 +540,8 @@ macro_rules! bit_struct {
         impl $crate::BitStruct<{$(<$actual as $crate::ValidCheck<$kind>>::ALWAYS_VALID &&)* true}> for $name {
             type Kind = $kind;
 
+            /// # Safety
+            /// - This is implemented automatically by the bit-struct crate.
             unsafe fn from_unchecked(inner: $kind) -> Self {
```

Comment:
Safety comments on unsafe functions should address under what conditions it is safe to call the function (and same for below unsafe functions).

Reply:
oops yup I messed up here.

Reply:
d146e84

Reply:
LGTM (I think github doesn't let me mark this thread as resolved for some reason)